website/docs/api/architectures.mdx

```diff
@@ -481,6 +481,280 @@ The other arguments are shared between all versions.
 
 </Accordion>
 
+## Curated transformer architectures {id="curated-trf",source="https://github.com/explosion/spacy-curated-transformers/blob/main/spacy_curated_transformers/models/architectures.py"}
+
+The following architectures are provided by the package
+[`spacy-curated-transformers`](https://github.com/explosion/spacy-curated-transformers).
+See the [usage documentation](/usage/embeddings-transformers#transformers) for
+how to integrate the architectures into your training config.
+
+### spacy-curated-transformers.AlbertTransformer.v1
+
+Construct an ALBERT transformer model.
+
+| Name                           | Description                                                                              |
```

Comment:
I'm not sure how useful that would be as the concept of canonical hyper parameters is somewhat unclear in this context? We do plan on adding a new CLI command to fill in those values based on the model we're loading from Hugging Face Hub (which would be the intended usecase anyway).

Reply:
This table refers to `spacy-curated-transformers.AlbertTransformer.v1`, right? So if you have that in a config file and you don't specify all parameters, and you use `init fill-config` it will pull in those defaults, no?

Reply:
That's correct (cf. https://github.com/explosion/spacy-curated-transformers/pull/16).

Reply:
Right, so then I don't really understand why it wouldn't be useful to document these?

Reply:
(resolving after internal discussion)